docs/rules.md

```diff
@@ -49,6 +49,12 @@ hide:
     11. Do not write or install software on computers belonging to other players or servers without their informed consent.
         1.  When advertising malicious software, it should be made very clear that it is malicious during every interaction.
     12. Do not use No Chat Reports' chat encryption feature in public chat. Using it in private messages is allowed however, as long as the receiving party is able to decode it.
+    13. Any form of in game gambling must be fair to the players.
+        1. Gambling may *never* be done with currency purchased with money that has value in the real world (FIAT, Crypto, etc.)
+        2. Any odds must be clearly displayed on an ingame monitor, and in the user interface where you gamble.
+        3. When a gambling operator chooses to not use Kromer, regular items or Krist directly in the gambling machines, the following rules are applicable.
+            1. The exchange rate for the coins to ingame currency must be static and can not be changed.
```

Comment:
This should only apply when there is a known/given exchange rate. But it should probably be allowed to change(within reason), because irl gambling money changes value too.